extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -1054,6 +1054,7 @@ describe('amp-analytics', function() {
 
     check('abc', 'abc', '');
     check('client id', 'client id', '');
+    check('client id\nand something', 'client id\nand something', '');
```

Comment:
Can we add one with `client id\nclientId()`, confirming that we don't get an args match?